react/NestedSelect/styles.styl

```diff
@@ -13,9 +15,17 @@
 .Row__selected
     font-weight bold
 
+.Row__caption
+    font-weight normal
+
 .Modal__title
     border-bottom 1px solid var(--silver)
     margin-bottom 0
+    padding-top 1rem
+    font-size $h2-font-size
+
+.Modal__title h2
+    font-size $h2-font-size
```

Comment:
Why do we need to specify h2 font size twice here?

Reply:
No need, the one not in the h2 was extraneous ! Thanks